### PR TITLE
Ml add hover effect to buttons

### PR DESCRIPTION
**Pull request by marklohr**

### CODE REVIEW

File: app/models/visit_group.rb [hunk 1]
```diff
@@ -35,7 +35,6 @@ class VisitGroup < ActiveRecord::Base
   attr_accessible :window_before
   attr_accessible :window_after
   acts_as_list scope: :arm
-  scope :at_position, ->(position) { where(position: position) }
```
Comment (wtholt) [inline app/models/visit_group.rb]: Why did you remove this query?

Comment (marklohr) [inline app/models/visit_group.rb]: I meant to remove this on my previous story / branch (PR #278) which was already merged in.  It is not used anywhere (see first two commit messages)...   I probably should have done this straight to master branch.

File: config/locales/en.yml [hunk 2]
```diff
@@ -105,14 +105,14 @@ en:
 
   # Bottom navigation
   bottom_navigation:
-    cancel: "Cancel"
-    continue: "Continue"
+    cancel: " ⟵ Cancel"
```
Comment (marklohr) [inline config/locales/en.yml]: Swapped out arrow .png files with unicode arrows.  Allows for better grouping during the hover state effect.  Loads faster than image file.
